ci/functional/test_main.sh

```diff
@@ -33,7 +33,7 @@ mkdir -p install/lib/daos/TESTING/ftest/avocado/job-results
 if $TEST_RPMS; then
     # shellcheck disable=SC2029
     ssh -i ci_key -l jenkins "${first_node}" \
-      "TEST_TAG=$test_tag                            \
+      "TEST_TAG=\"$test_tag\"                        \
        TNODES=$tnodes                                \
        FTEST_ARG=$FTEST_ARG                          \
```

Comment:
This will probably be covered in another PR, but ideally we should be doing the following:
```suggestion
      "TEST_TAG=\"$test_tag\"                        \
       TNODES=\"$tnodes\"                            \
       FTEST_ARG=\"$FTEST_ARG\"                      \
```

Reply:
Done.